src/components/Pagination.js

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import { Pagination as BSPagination } from 'react-bootstrap';
+
+export const Pagination = ({
+  items,
+  maxButtons,
+  boundaryLinks,
+  activePage,
+  onSelect,
+  ...props
+}) => {
+  function onChange(event) {
+    let text = event.target.text;
+    let pageNumber = 1;
+    if (text == null) {
+      pageNumber = activePage;
+    } else {
+      pageNumber = parseInt(text);
+    }
+
+    if (pageNumber !== activePage) {
+      onSelect(pageNumber);
+    }
+  }
+
+  let half = maxButtons / 2;
+  var min = activePage - half;
+  if (min < 1) {
+    min = 1;
+  }
+  var max = activePage + half;
+  if (max > items) {
+    max = items;
+  }
+
+  if (activePage - min < half) {
+    max = max + (half - (activePage - min));
+  }
+
+  if (max - activePage < half) {
+    min = min - (half - (max - activePage)) + 1;
+  }
+
+  if (min < 1) {
+    min = 1;
+  }
+  if (max > items) {
+    max = items;
+  }
+
+  let pageNumbers = [];
+  pageNumbers.push(
+    <BSPagination.Item active={1 === activePage}>1</BSPagination.Item>
+  );
+  if (min > 2) {
+    pageNumbers.push(<BSPagination.Ellipsis />);
+  }
+  for (var i = min; i < max; ++i) {
+    if (i === 1 || i === items) {
+      continue;
+    }
+    pageNumbers.push(
+      <BSPagination.Item active={i === activePage}>{i}</BSPagination.Item>
+    );
+  }
+  if (items - max > 0) {
+    pageNumbers.push(<BSPagination.Ellipsis />);
```

Comment:
After a hard refresh (cmd-shift-r) of the page (or first load), clicking on the ellipsis in the UI for the sample person data yields an error involving the Redux layer.  It does not seem to happen on subsequent navigation back to the search results.

![image](https://user-images.githubusercontent.com/6333668/84788328-42500e80-afac-11ea-988d-8494e04e98ff.png)